### ISSUE TITLE
last_mq_len is never updated? [JIRA: RIAK-1825]

### ISSUE THREAD

**Issue by DeadZen** [basho/sidejob#11]
https://github.com/basho/sidejob/blob/develop/src/sidejob_worker.erl#L223
This field is calculated based on a last value that is never actually updated.

**Comment by engelsanchez** [comment]
@DeadZen you are right.  When not using a custom module to update those values, that value is never updated.  It seems the logic has been like this since the beginning.  We'll clean it up.

**Comment by DSomogyi** [comment]
.
